test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -577,5 +577,15 @@ public void It_passes_ridless_target_to_compiler()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void It_restores_only_ridless_tfm()
```

Comment:
OK figured it out. For some reason, the multiple rid problem only repros on apps targeting `netcoreapp1.0`, not on libraries targeting `netstandard1.5`. I guess some combination of the conditions in Microsoft.NuGet.targets is only affecting these projects, but its not obvious which ones. (To confirm, New NetStandard Library -> Add System.ValueTuple, does not produce any warnings.)

I've moved the test to `GivenThatWeWantToBuildANetCoreApp` and modified it to check values from the `HelloWorld` app. Now `build` passes, but `build -FullMSBuild` fails. I'll push this change to confirm Jenkins fails, then disable the test until we update Jenkins.